CONTRIBUTING.md

```diff
@@ -50,19 +50,50 @@ To ensure that all components function correctly in both Marko v3 and v4, we run
 
 ## Pull Requests
 
-Please ensure an issue exists in GitHub before writing any code. Every pull request must have an associated issue for tracking and triage purposes.
+This section contains information and in-house etiquette regarding pull requests (or "PRs" as they are often called).
```

Comment:
Is this limited to "in-house" etiquette?

Reply:
I guess I meant "house rules". But I can probably remove this word to avoid confusion.